scripts/grunt/options/chromium.js

```diff
@@ -0,0 +1,72 @@
+
+
+module.exports = function (config, grunt) {
+  'use strict';
+
+  const request = require('request-promise-native');
+  const extract = require('extract-zip');
+  const fs = require('fs');
+  const mkdirp = require('mkdirp');
+  const os = require('os');
+  const path = require('path');
+
+  function extractZip(file, dir) {
+    return new Promise(fulfill => extract(file, { dir: path.resolve(dir) }, fulfill));
+  }
+
+  grunt.registerTask('chromium', 'Download chromium to tools', function () {
+
+    // make task async
+    var done = this.async();
+
+    const chromium_revision = grunt.file.readJSON('./package.json').grafana.chromium_revision;
+    const dest = `./tools/chromium/${chromium_revision}`;
+    const downloadURLs = {
+      linux: `https://storage.googleapis.com/chromium-browser-snapshots/Linux_x64/${chromium_revision}/chrome-linux.zip`,
+      mac: `https://storage.googleapis.com/chromium-browser-snapshots/Mac/${chromium_revision}/chrome-mac.zip`,
+      win32: `https://storage.googleapis.com/chromium-browser-snapshots/Win/${chromium_revision}/chrome-win32.zip`,
+      win64: `https://storage.googleapis.com/chromium-browser-snapshots/Win_x64/${chromium_revision}/chrome-win32.zip`,
+    };
+
+    var platform = os.platform();
```

Comment:
As it is, you would end up downloading Linux_x64 version on Linux armv6/v7/aarch64. It would be better to avoid the download.

Reply:
will fix